vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -28,13 +30,16 @@ def __init__(self, model_xml: bytes,
             extensions
         """
         super().__init__()
-        from openvino.inference_engine import IECore
+        # Convert from the vcap device naming format to openvino format
+        device_name = "CPU" if device_name[:4] == "CPU:" else device_name
+
+        from openvino.inference_engine import \
+            IECore, ExecutableNetwork, IENetwork, StatusCode
 
         self.ie = IECore()
-        if cpu_extensions is None:
 
-            extension_path = os.environ.get("OPENVINO_EXTENSION_PATH",
-                                            None)
+        if cpu_extensions is None:
+            extension_path = os.environ.get("OPENVINO_EXTENSION_PATH", None)
```

Comment:
Do you think we should keep CPU extension support for now?

Reply:
I remember there being some discussion on removing it.

Reply:
I decided against it, then I saw your comment, and I was convinced yet again. 

What I did is I removed cpu_extensions, and added the (optional) ability to pass in your own custom IECore. This might enable people to prepare it however they'd like, including loading plugins, extensions, or whatever else.